spec.bs

```diff
@@ -2792,6 +2836,22 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+The {{Fence/notifyEvent}} method of the {{Fence}} interface introduces a new event type, `fencedtreeclick`.
+
+The [=event handlers on elements, Document objects, and Window objects=] section of [[HTML]] is modified to 
```

Comment:
This should not be a definition it should just be a manual anchor link to that section.

Reply:
Done.